### DISCUSSION TITLE
Activating #define FAN_KICKSTART_TIME in precompiled bins

### DISCUSSION THREAD

**Discussion by Fury1SOG** [Jyers/Marlin#312] [Q&A] [answered]
Hello,
First of all, thank you very much for this firmware. It has greatly increased the functionality and QoL of my printer!

I recently replaced the part cooling fan in my E3V2 and it is now mounted in a more visible location than the stock fan.
I've noticed that the fan has trouble starting at low rpms, such as when you are printing PETG.

Configuration_adv.h has a setting called #define FAN_KICKSTART_TIME that boosts the fan rpm for a specified amount of time to get them to start at low rpms.
Would it be possible to enable this in the precompiled bins? Thank you!

**Comment by Jyers** [comment, marked as answer]
If it doesn't have drawbacks for other setups and there is some interest I could definitely enable this. I'll look into it.

**Reply by Fury1SOG** [reply]
Thank you very much, Jyers! 
I suspect that a lot of people don't realize that their part cooling fan isn't turning on according to their slicer settings because it is hidden away behind a cover.
